iothub/service/samples/getting started/InvokeDeviceMethod/Parameters.cs

```diff
@@ -27,20 +27,5 @@ internal class Parameters
             HelpText = "The Id of the device to receive the direct method." +
             "\nDefaults to 'MyDotnetDevice'.")]
         public string DeviceId { get; set; } = "MyDotnetDevice";
-
-        public static void ValidateConnectionString(string hubConnectionString)
-        {
-            try
-            {
-                _ = IotHubConnectionStringBuilder.Create(hubConnectionString);
```

Comment:
@drwill-ms This class is no longer public, so we can't use this validation in here directly.  We have an "IotHubConnectionStringParser" that we could make public and comment that it is only used for validation. The user isn't ever required to interact with this type otherwise though.

Reply:
Just validate it isn't null or whitespace and then rely on the client init to validate it the remainder.

Reply:
We don't make it required via CommandLine [Option] attribute, because it can be set via environment variable too. So checking it here is simply meant to ensure the user passed something.

Reply:
Updated